static/css/profile.css

```diff
@@ -42,8 +41,17 @@ main > header {
 .profile-intro section {
   flex-basis: 100%;
   order: 2;
+  margin-top: 5rem;
+}
+.profile-intro section  a {
+  display: block;
   margin-top: 2rem;
 }
+
+article > a:not(.read) {
```

Comment:
That was moved to a [specific articles list css](https://github.com/neomad-team/neomad.org/blob/master/static/css/_article_list.css#L17)